doc/changelog/03-notations/12163-fix-12159.rst

```diff
@@ -0,0 +1,11 @@
+- **Fixed:**
+  Numeral Notations now play better with multiple scopes for the same
+  inductive. Previously, when multiple numeral notations where defined
+  for the same inductive, only the last one was considered for
+  printing. Now, among the printers that produce an output and have a
+  scope delimiter (if required), the selection is made according to
+  open scopes (or the last defined one if no appropriate scope is
+  open)
```

Comment:
I think this is a bit clearer and more explicit:
```suggestion
  printing. Now, among the printers that produce an output and either
  have a scope delimiter or are open, the selection is made according
  to the order of opened scopes, or according to the last defined
  scope if no appropriate scope is open
```

Reply:
Thanks, done (just replaced scopes by notation for the last default choice).